_pages/about-us/teams/products-and-platforms.md

```diff
@@ -15,13 +17,13 @@ Products and Platforms is one of the 5 business units within 18F. We strive to i
 
 * 18F's work on the [eRegs platform](https://eregs.github.io/) with various partners.
 
-* With partners in TTS, the [Draft US Web Design Standards](https://standards.usa.gov/): open-source UI components and visual styles to create beautiful, consistent experiences across federal government websites.
```

Comment:
I think we landed on hyperlinking TTS (http://www.gsa.gov/portal/category/25729) whenever it's mentioned in the handbook.  Does that work for y'all?